src/main/java/com/slack/kaldb/elasticsearchApi/searchRequest/EsSearchRequest.java

```diff
@@ -0,0 +1,106 @@
+package com.slack.kaldb.elasticsearchApi.searchRequest;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.collect.Lists;
+import com.slack.kaldb.elasticsearchApi.searchRequest.aggregations.SearchRequestAggregation;
+import com.slack.kaldb.proto.service.KaldbSearch;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class EsSearchRequest {
```

Comment:
Since these classes are not exposed outside this package, let's make them package private.

Reply:
These are actually used outside of the package via the `ElasticsearchApiService`, so to change the access modifier we would need to reorganize the package structure.